### PR TITLE
docs (appium): Add "Write a Test" Java tutorial

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Linking https://github.com/appium/appium/issues/16799

File: packages/appium/docs/en/quickstart/test-java.md [hunk 1]
```diff
@@ -2,4 +2,17 @@
 title: Write a Test (Java)
 ---
 
-TODO
+The Appium team maintains an official [client](https://github.com/appium/java-client) for Java programming language.
```
Comment (jlipps) [inline packages/appium/docs/en/quickstart/test-java.md:5]:
```suggestion
The Appium team maintains an official [client](https://github.com/appium/java-client) for the Java programming language.
```

File: packages/appium/docs/en/quickstart/test-java.md [hunk 2]
```diff
@@ -2,4 +2,17 @@
...
+It is built on top of [Selenium](https://github.com/SeleniumHQ/selenium)
```
Comment (jlipps) [inline packages/appium/docs/en/quickstart/test-java.md:6]:
```suggestion
It is built on top of [Selenium](https://github.com/SeleniumHQ/selenium).
```

File: packages/appium/docs/en/quickstart/test-java.md [hunk 3]
```diff
@@ -2,4 +2,17 @@
...
+lib. You could also use this client in your Kotlin projects.
```
Comment (jlipps) [inline packages/appium/docs/en/quickstart/test-java.md:7]:
```suggestion
You can also use this client in your Kotlin projects.
```

File: packages/appium/docs/en/quickstart/test-java.md [hunk 4]
```diff
@@ -2,4 +2,17 @@
...
+
+Follow the [Add Appium java client to your test framework](https://github.com/appium/java-client#add-appium-java-client-to-your-test-framework)
+tutorial in order to connect the library to your test framework sources.
+
+Appium java client has dedicated classes to support most of the official Appium drivers. For other drivers
```
Comment (jlipps) [inline packages/appium/docs/en/quickstart/test-java.md:12]:
```suggestion
The Appium Java client has dedicated classes to support most of the official Appium drivers. For other drivers
```

File: packages/appium/docs/en/quickstart/test-java.md [hunk 5]
```diff
@@ -2,4 +2,17 @@
...
+you could simply use [AppiumDriver](https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/AppiumDriver.java) class
```
Comment (jlipps) [inline packages/appium/docs/en/quickstart/test-java.md:13]:
```suggestion
you could simply use the [AppiumDriver](https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/AppiumDriver.java) class
```

File: packages/appium/docs/en/quickstart/test-java.md [hunk 6]
```diff
@@ -2,4 +2,17 @@
...
+or build your custom derivatives from it. Check the [Drivers Support](https://github.com/appium/java-client#drivers-support)
+article to know more details about currently present driver class implementations.
```
Comment (jlipps) [inline packages/appium/docs/en/quickstart/test-java.md:15]:
```suggestion
article to learn more about the current driver class implementations.
```

File: packages/appium/docs/en/quickstart/test-java.md [hunk 7]
```diff
@@ -2,4 +2,17 @@
...
+
+Follow the [Usage Examples](https://github.com/appium/java-client#usage-examples) article in order understand
+how to invoke java client features from your test framework.
```
Comment (jlipps) [inline packages/appium/docs/en/quickstart/test-java.md:18]:
```suggestion
how to invoke Java client features from your test framework.
```

**Review by jlipps — APPROVED** [review]

**Pull request merged**